src/components/actionBar/filterSort/FilterSort.tsx

```diff
@@ -1,3 +1,261 @@
+                    open={openBusTypeFilter}
+                    MenuListProps={{ 'aria-labelledby': 'bus-type-button' }}
+                    onClose={handleMenuClose}
+                >
+                    {/* <FormControl> */}
+                    <RadioGroup
+                        onChange={busFilterHandler}
+                        value={busTypeFilter}
+                    >
+                        <StyledFormControlLabel
+                            label="AC"
+                            control={<Radio value="AC" icon={<AcUnit />} />}
+                        />
+                        <StyledFormControlLabel
+                            label="Non-AC"
+                            control={<Radio value="Non-AC" icon={<Air />} />}
+                        />
+                    </RadioGroup>
+                    {/* </FormControl> */}
+                    <Button
+                        onClick={() => {
+                            setBusTypeFilter(null);
+                        }}
+                    >
+                        Clear
+                    </Button>
+                </Menu>
+                <Menu
+                    id="seat-type-menu"
+                    anchorEl={anchorElSeat}
+                    open={openSeatTypeFilter}
+                    MenuListProps={{ 'aria-labelledby': 'seat-type-button' }}
+                    onClose={handleMenuClose}
+                >
+                    <RadioGroup
+                        onChange={seatFilterHandler}
+                        value={seatTypeFilter}
+                    >
+                        <StyledFormControlLabel
+                            label="Seater"
+                            control={
+                                <Radio
+                                    value="Seater"
+                                    icon={<AirlineSeatReclineNormal />}
+                                />
+                            }
+                        />
+                        <StyledFormControlLabel
+                            label="Sleeper"
+                            control={<Radio value="Sleeper" icon={<Hotel />} />}
+                        />
+                    </RadioGroup>
+                    <Button
+                        onClick={() => {
+                            setSeatTypeFilter(null);
+                        }}
+                    >
+                        Clear
+                    </Button>
+                </Menu>
+
+                {/* Sort menu */}
+                <Menu
+                    id="sort-menu"
+                    anchorEl={anchorElSort}
+                    open={openSort}
+                    MenuListProps={{ 'aria-labelledby': 'sort-button' }}
+                    onClose={handleMenuClose}
+                >
+                    <RadioGroup onChange={sortHandler} value={sortBy}>
+                        <StyledFormControlLabel
+                            label="Start date"
```

Comment:
All of these labels could be updated with translations.

Reply:
All labels have been replaced using usetraslation.

Reply:
Toggle functionality and Passenger count validation using regex will be covered during functionality phase.